release/data_processing_tests/multi_node.yaml

```diff
@@ -108,7 +108,8 @@ setup_commands:
     # - sudo apt install -y gdb
     # - sudo apt install -y linux-tools-common
     - echo 'export PATH="$HOME/anaconda3/envs/tensorflow_p36/bin:$PATH"' >> ~/.bashrc
-    - pip install -U https://s3-us-west-2.amazonaws.com/ray-wheels/test_wheels/latest/ray-2.0.0.dev0-cp36-cp36m-manylinux2014_x86_64.whl
+    - pip uninstall -y ray
+    - pip install -U https://s3-us-west-2.amazonaws.com/ray-wheels/latest/ray-2.0.0.dev0-cp36-cp36m-manylinux2014_x86_64.whl
```

Comment:
This is a non e2e release test, right? As in, it is started manually and uses the Ray OSS autoscaler?

Reply:
It is not now. I will do that once shuffle test results are properly reported.